03-Running AzTS solution from UI/README.md

```diff
@@ -70,6 +71,27 @@ Here are some additional video tutorials explaining the features provided by AzT
 ### **How to export control scan logs to local machine**
 [![UI](../Images/04_UI_ExportToCSV.gif)](https://azsk-azts-cdn.azureedge.net/videosforpublicgithubdoc/Ext_Export_To_CSV.mp4)
 
+## Miscellanoues UI features
+- ### **Scan In Progress:**
+
+AzTS UI can be used to scan subscriptions(s) manually. Once such manusal scan is triggered, you can get notification about scan progress status (e.g. Scan In Progress or Scan completed) by enabling Scan In Progress feature in AzTS UI. 
```

Comment:
Instead of manually, we can say on demand scan. Thoughts?

Reply:
makes sense, updated, thanks!!